public/script.js

```diff
@@ -7510,6 +7510,7 @@ export function showSwipeButtons() {
 
     //allows for writing individual swipe counters for past messages
     $('.last_mes .swipes-counter').text(swipeCounterText);
```

Comment:
You can combine calls to prevent double querying
```suggestion
    $('.last_mes .swipes-counter').text(swipeCounterText).show();
```